<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PlaybackUI</class>
 <widget class="QWidget" name="PlaybackUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>408</width>
    <height>148</height>
   </rect>
  </property>
  <property name="focusPolicy">
   <enum>Qt::NoFocus</enum>
  </property>
  <property name="windowTitle">
   <string>Playback Controls</string>
  </property>
  <property name="styleSheet">
   <string notr="true">	background: rgb(51, 51, 51);</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="1">
    <widget class="QLabel" name="timeDescripterLabel">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="styleSheet">
      <string notr="true">color: white;</string>
     </property>
     <property name="text">
      <string>Time:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QPushButton" name="openButton">
     <property name="styleSheet">
      <string notr="true">QPushButton{
padding: 9px;
margin: 5px;
color: rgb(255,255,255);
border: 1px solid rgb(45,45,45);
background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 rgb(140,140,140), stop: 1 rgb(100,100,100));
}
QPushButton:hover{
padding: 3px;
padding-top:5px;
margin: 5px;
color: rgb(255,255,255);
border: 1px solid rgb(45,45,45);
background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 rgb(160,160,160), stop: 1 rgb(120,120,120));
}
QPushButton:pressed{
padding: 3px;
padding-top:5px;
margin: 5px;
color: rgb(255,255,255);
border: 1px solid rgb(45,45,45);
background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 rgb(120,120,120), stop: 1 rgb(80,80,80));
}</string>
     </property>
     <property name="text">
      <string>Open</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QLabel" name="playbackTimeLabel">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QPushButton" name="exitButton">
     <property name="styleSheet">
      <string notr="true">QPushButton{
padding: 9px;
margin: 5px;
color: rgb(255,255,255);
border: 1px solid rgb(45,45,45);
background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 rgb(140,140,140), stop: 1 rgb(100,100,100));
}
QPushButton:hover{
padding: 3px;
padding-top:5px;
margin: 5px;
color: rgb(255,255,255);
border: 1px solid rgb(45,45,45);
background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 rgb(160,160,160), stop: 1 rgb(120,120,120));
}
QPushButton:pressed{
padding: 3px;
padding-top:5px;
margin: 5px;
color: rgb(255,255,255);
border: 1px solid rgb(45,45,45);
background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 rgb(120,120,120), stop: 1 rgb(80,80,80));
}</string>
     </property>
     <property name="text">
      <string>Exit</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1" colspan="2">
    <widget class="QSlider" name="playbackPositionSlider">
     <property name="styleSheet">
      <string notr="true">QSlider::groove:horizontal {
border: 1px solid #bbb;
background: white;
height: 10px;
border-radius: 4px;
}

QSlider::sub-page:horizontal {
background: qlineargradient(x1: 0, y1: 0,    x2: 0, y2: 1,
    stop: 0 #aaa, stop: 1 #777);
background: qlineargradient(x1: 0, y1: 0.2, x2: 1, y2: 1,
    stop: 0 #aaa, stop: 1 #777);
border: 1px solid #777;
height: 10px;
border-radius: 4px;
}

QSlider::add-page:horizontal {
background: #fff;
border: 1px solid #777;
height: 10px;
border-radius: 4px;
}

QSlider::handle:horizontal {
background: qlineargradient(x1:0, y1:0, x2:1, y2:1,
    stop:0 #eee, stop:1 #ccc);
border: 1px solid #777;
width: 13px;
margin-top: -2px;
margin-bottom: -2px;
border-radius: 4px;
}

QSlider::handle:horizontal:hover {
background: qlineargradient(x1:0, y1:0, x2:1, y2:1,
    stop:0 #fff, stop:1 #ddd);
border: 1px solid #444;
border-radius: 4px;
}

QSlider::sub-page:horizontal:disabled {
background: #bbb;
border-color: #999;
}

QSlider::add-page:horizontal:disabled {
background: #eee;
border-color: #999;
}

QSlider::handle:horizontal:disabled {
background: #eee;
border: 1px solid #aaa;
border-radius: 4px;
}R</string>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
